api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/security/SecurityConfiguration.java

```diff
@@ -95,26 +95,24 @@ public class FilterChainBasicAuthOrTokenOrCertForApiDoc {
         @Bean
         public SecurityFilterChain basicAuthOrTokenOrCertApiDocFilterChain(HttpSecurity http) throws Exception {
             mainframeCredentialsConfiguration(
-                baseConfiguration(http.requestMatchers().antMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE).and())
+                    baseConfiguration(http.requestMatchers(matchers -> matchers.antMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE)))
             )
-                .authorizeRequests()
-                .antMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE).authenticated()
-                .and()
-                .authenticationProvider(gatewayLoginProvider)
-                .authenticationProvider(gatewayTokenProvider)
-                .authenticationProvider(new CertificateAuthenticationProvider());
+                    .authorizeRequests(requests -> requests
+                            .antMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE).authenticated())
+                    .authenticationProvider(gatewayLoginProvider)
+                    .authenticationProvider(gatewayTokenProvider)
+                    .authenticationProvider(new CertificateAuthenticationProvider());
 
             if (verifySslCertificatesOfServices || !nonStrictVerifySslCertificatesOfServices) {
                 if (isAttlsEnabled) {
-                    http.x509()
-                        .userDetailsService(x509UserDetailsService())
-                        .and()
-                        .addFilterBefore(reversedCategorizeCertFilter(), X509AuthenticationFilter.class)
-                        .addFilterBefore(new AttlsFilter(), X509AuthenticationFilter.class)
-                        .addFilterBefore(new SecureConnectionFilter(), AttlsFilter.class);
+                    http.x509(x509 -> x509
+                            .userDetailsService(x509UserDetailsService()))
+                            .addFilterBefore(reversedCategorizeCertFilter(), X509AuthenticationFilter.class)
+                            .addFilterBefore(new AttlsFilter(), X509AuthenticationFilter.class)
+                            .addFilterBefore(new SecureConnectionFilter(), AttlsFilter.class);
                 } else {
-                    http.x509()
-                        .userDetailsService(x509UserDetailsService());
+                    http.x509(x509 -> x509
```

Comment:
Is there any reason to use lambda here?

Reply:
I was just applying some automatic detection in the IDE, but I can roll it back if not ok